src/engraving/rendering/dev/horizontalspacing.cpp

```diff
@@ -608,6 +610,17 @@ KerningType HorizontalSpacing::computeNoteKerningType(const Note* note, const En
     return KerningType::KERNING;
 }
 
+KerningType HorizontalSpacing::computeNoteDotKerningType(const NoteDot* noteDot, const EngravingItem* item2)

```

Comment:
I don't think it's necessary to make special function for this. Unless I'm mistaken, if you just add `ElementType::NOTE_DOT` to the function `isSameVoiceKerningLimited` you'll get the same result more simple and more efficiently.